src/js/autocomplete.js

```diff
@@ -1,14 +1,264 @@
+// We use our own fork of accessible-autocomplete because of this bug -- https://github.com/alphagov/accessible-autocomplete/issues/210#issuecomment-845860595
+// Our fork uses this change -- https://github.com/alphagov/accessible-autocomplete/pull/491
+// If the bug is fixed in accessible-autocomplete then we can stop using our fork
```

Comment:
We should also include this bug fix -- https://github.com/alphagov/accessible-autocomplete/pull/496

Reply:
This was done in [`0f25101` (#2)](https://github.com/Financial-Times/o-autocomplete/pull/2/commits/0f2510189a8853e789e4616fbae7b7d4797e479d)